apps/web/src/server/db/index.ts

```diff
@@ -0,0 +1,12 @@
+import { Client } from "@planetscale/database";
+import { drizzle } from "drizzle-orm/planetscale-serverless";
+
+import { env } from "~/env.mjs";
+import * as schema from "./schema";
+
+export const db = drizzle(
+  new Client({
+    url: env.DATABASE_URL,
+  }).connection(),
+  { schema }
+);
```

Comment:
<i class='graphite__hidden'>[Re: lines 1 to 13]<br/></i>

move to db<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/AbhiShake1/titeet/1?utm_source=unchanged-line-comment">Graphite</a>.</span>